testdata/java/mistakes/openingBracketMissing/src/test/java/com/eval/OpeningBracketMissingTest.java

```diff
@@ -8,7 +8,7 @@ public class OpeningBracketMissingTest {
 	public void openingBracketMissing1() {
 		int x = -1;
 		int expected = -1;
-		int actual = OpeningBracketIsMissing.openingBracketMissing(x);
+		int actual = OpeningBracketMissing.openingBracketMissing(x);
 
 		assertEquals(expected, actual);
 	}
```

Comment:
Should we maybe add some tests that all repositories have to build?

Reply:
They'll fail the build since they all have compilation errors

Reply:
ah yes... well of course they do :face_in_clouds: